### PR TITLE
Tcpdump process terminate exceptly

### PR DESCRIPTION
**Pull request by yanglei-rh**
ID:1923955

Signed-off-by: Lei Yang <leiayang@redhat.com>

### CODE REVIEW

**Comment by yanglei-rh** [comment]
@PaulYuuu @kimi1978wy  Please help me review it, thanks a lot.

**Review by kimi1978wy — COMMENTED** [review]
@yanglei-rh I hit this error
(1/3) Host_RHEL.m8.u4.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.Win10.i386.io-github-autotest-qemu.virtual_nic.private.q35: ERROR: 'list' object has no attribute 'cmd_status_output' (277.67 s)
 (2/3) Host_RHEL.m8.u4.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.Win10.x86_64.io-github-autotest-qemu.virtual_nic.private.q35: ERROR: 'list' object has no attribute 'cmd_status_output' (255.91 s)
 (3/3) Host_RHEL.m8.u4.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.Win2016.x86_64.io-github-autotest-qemu.virtual_nic.private.q35: ERROR: VM 'vm1' failed to start: Error occurred while executing make_create_command(). Check the log for traceback. (5.78 s)

**Comment by yanglei-rh** [comment]
Test Pass:
(2/2) Host_RHEL.m8.u4.product_rhel.qcow2.virtio_scsi.up.virtio_net.Guest.Win2019.x86_64.io-github-autotest-qemu.virtual_nic.private.q35: PASS (594.01 s)
(2/2) Host_RHEL.m8.u4.product_rhel.qcow2.virtio_scsi.up.virtio_net.Guest.Win10.i386.io-github-autotest-qemu.virtual_nic.private.q35: PASS (361.30 s)
(1/1) Host_RHEL.m8.u4.product_rhel.qcow2.virtio_scsi.up.virtio_net.Guest.Win10.x86_64.io-github-autotest-qemu.virtual_nic.private.q35: PASS (335.16 s)

**Comment by yanglei-rh** [comment]
Hi @kimi1978wy I have resolved the error, please help me review it again, thanks.

**Comment by kimi1978wy** [comment]
LGTM

Acked by: Yu Wang <wyu@redhat.com>

 (4/6) Host_RHEL.m8.u4.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.Win2016.x86_64.io-github-autotest-qemu.virtual_nic.private.q35: PASS (266.18 s)
 (5/6) Host_RHEL.m8.u4.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.Win8.i386.1.io-github-autotest-qemu.virtual_nic.private.q35: PASS (113.60 s)
 (6/6) Host_RHEL.m8.u4.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.Win8.x86_64.1.io-github-autotest-qemu.virtual_nic.private.q35: PASS (134.67 s)

File: qemu/tests/virtual_nic_private.py [hunk 1]
```diff
@@ -68,6 +85,20 @@ def data_mon(session, cmd, timeout):
...
+            install_wireshark_cmd = params.get("install_wireshark_cmd")
+            install_wireshark_cmd = utils_misc.set_winutils_letter(
+                    sessions[2], install_wireshark_cmd)
+            status, output = sessions[2].cmd_status_output(install_wireshark_cmd,
+                                                           timeout=timeout)
+            if status:
+                test.error("Failed to install wireshark, status=%s, output=%s"
+                           % (status, output))
+            logging.info("Wait for wireshark installation to complete")
+            utils_misc.wait_for(
```
Comment (PaulYuuu) [inline qemu/tests/virtual_nic_private.py:98]: Is `c:\${wireshark_name} /S` running in the background so need to continue to check the installer at the next step?

Comment (yanglei-rh) [reply qemu/tests/virtual_nic_private.py:98]: Hi @PaulYuuu  I think this is necessary. Check “{wireshark_name}” is actually to give some installation time.

Comment (PaulYuuu) [reply qemu/tests/virtual_nic_private.py:98]: Could you get the output of `c:\${wireshark_name} /S`, if it runs in the background, then the next step is necessary.

Comment (yanglei-rh) [reply qemu/tests/virtual_nic_private.py:98]: Hi @PaulYuuu I confirmed that “c:\${wireshark_name} /S“ is running in the background. And the output is empty in the test. So I think the next step is necessary.

Comment (PaulYuuu) [reply qemu/tests/virtual_nic_private.py:98]: OK, thanks for the explanation.

**Review by PaulYuuu — APPROVED** [review]

**Pull request merged**